src/query/service/src/servers/mysql/mysql_federated.rs

```diff
@@ -147,121 +212,157 @@ impl MySQLFederated {
+            // "(?i)^(LOCK TABLES FOR BACKUP)"
+            None,
+            // "(?i)^(UNLOCK BINLOG(.*))"
+            None,
+            // "(?i)^(/\\*!40101 SET(.*) \\*/)$"
+            None,
             // DBeaver.
-            ("(?i)^(SHOW WARNINGS)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW WARNINGS)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW PLUGINS)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW COLLATION)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW CHARSET)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW ENGINES)", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SELECT @@(.*))", None),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW @@(.*))", None),
-            (
-                "(?i)^(/\\* ApplicationName=(.*)SET net_write_timeout(.*))",
-                None,
-            ),
-            (
-                "(?i)^(/\\* ApplicationName=(.*)SET SQL_SELECT_LIMIT(.*))",
-                None,
-            ),
-            ("(?i)^(/\\* ApplicationName=(.*)SHOW VARIABLES(.*))", None),
+            // "(?i)^(SHOW WARNINGS)"
+            None,
+            // (?i)^(/\\* ApplicationName=(.*)SHOW WARNINGS)"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW PLUGINS)"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW COLLATION)"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW CHARSET)"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW ENGINES)"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SELECT @@(.*))"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW @@(.*))"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SET net_write_timeout(.*))"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SET SQL_SELECT_LIMIT(.*))"
+            None,
+            // "(?i)^(/\\* ApplicationName=(.*)SHOW VARIABLES(.*))"
+            None,
             // mysqldump 5.7.16
-            ("(?i)^(/\\*!40100 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40103 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40111 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40101 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40014 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40000 SET(.*) \\*/)$", None),
-            ("(?i)^(/\\*!40000 ALTER(.*) \\*/)$", None),
+            // "(?i)^(/\\*!40100 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40103 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40111 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40101 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40014 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40000 SET(.*) \\*/)$"
+            None,
+            // "(?i)^(/\\*!40000 ALTER(.*) \\*/)$"
+            None,
         ];
 
+        let rules = MIXED_CHECK_FEDERATED_REGEX_LIST
```

Comment:
It's ziped, so why not make `MIXED_CHECK_FEDERATED_REGEX_LIST` as `Vec<(Regex, Option<(TableSchemaRef, DataBlock)> )>`

Reply:
So we can save the comments